index.html

```diff
@@ -248,10 +246,59 @@ <h2 class="about_me_title">About me.</h2>
                 </li>
                 <li class="language">Database Mangement</li>
                 <li class="language">Version Control</li>
-                
+
                 <li class="language">CLI</li>
                 <li class="language">API Design</li>
                 <li class="language">Web Development</li>
             </ul>
   </div>
  </section>
+
+
+
+    <section id="form_container_link" class="form_section">
+      <div class="form_container">
+        <h3 class="form_text">I'm always interested in hearing about new projects, so if you'd like to chat please get in</h2>
+
+        <form id="form_contact" class="contact_me_form" action="https://formspree.io/f/mnqygakk" method="POST">
+
+            <input class="form_input_fnames" form="contact_form" id="name" name="name" type="text" placeholder="Full names" maxlength="30" required>
+            <hr>
+
+            <input class="form_input_lnames" form="contact_form" name= "last_name" type="text" placeholder="Last name" maxlength="30" required>
+            <hr class="lname_line">
+
+            <input class="form_input_email" id="email" name="email" form="contact_form" type="email" placeholder="Email address" required>
+            <hr>
+
+            <textarea class="form_input_textarea" id="msg" name="message" cols="30" rows="5" placeholder="  Write me something..."maxlength="500" required></textarea> <br>
```

Comment:
Let's update the placeholder text of the `textarea`  so as to match the template on Figma 👍 

Your design
![image](https://user-images.githubusercontent.com/78251927/228388802-232960ec-2d00-4491-8687-107c2829d07d.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/228388832-9ebdd9e9-c9cf-4a34-9dfd-94f2d378b3d8.png)